4-tools/manual.ipynb

```diff
@@ -25,7 +25,7 @@
     "\n",
     "[The Python Style Guide](https://www.python.org/dev/peps/pep-0008/), commonly referred to as PEP 8, was already mentioned in the first lesson. Here we introduce tools that allow you to check if your code is PEP 8 compliant. \n",
     "\n",
-    "## Ruff\n",
+    "## Ruff linter\n",
```

Comment:
Better to have a single section for Ruff than separate sections for Ruff linter and Ruff formatter.